Source/Tests/run_tests.py

```diff
@@ -1,4 +1,5 @@
 
+import argparse
```

Comment:
NI is standardizing on using `click` for the command line parser. However, for this simple case, I think argparse is okay. If you used `click`, you'd probably have to install it into the Python environment on the current test runner.